EmbeddingTestBeds/Embedding.Droid/Assets/AboutAssets.txt

```diff
@@ -0,0 +1,19 @@
+Any raw assets you want to be deployed with your application can be placed in
+this directory (and child directories) and given a Build Action of "AndroidAsset".
+
+These files will be deployed with you package and will be accessible using Android's
```

Comment:
Someone needs to fix that typo. `with you package`.

Reply:
Should be fixed now.